waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import styled from 'styled-components';
+import {ArrowIosDownwardOutline, ArrowIosUpwardOutline} from '@styled-icons/evaicons-outline';
+
+interface IProps {
+  open?: boolean;
```

Comment:
BTW how come you made this optional? Shouldn't the dropdown always just be either open or not open?